package.json

```diff
@@ -34,11 +34,11 @@
   },
   "private": true,
   "dependencies": {
-    "@alfresco/adf-content-services": "6.8.0-8186121470",
-    "@alfresco/adf-core": "6.8.0-8186121470",
-    "@alfresco/adf-extensions": "6.8.0-8186121470",
-    "@alfresco/eslint-plugin-eslint-angular": "6.8.0-8186121470",
-    "@alfresco/js-api": "7.7.0-8186121470",
+    "@alfresco/adf-content-services": "6.8.0-8137807425",
```

Comment:
are changes in this file intentional? I think you could get them by that issue with not yours commits